common/app/routes/challenges/components/classic/Classic.jsx

```diff
@@ -81,15 +99,30 @@ export class Challenge extends PureComponent {
     );
   }
 
+  keyBindings(event) {
+    const map = this.props.keyMap;
```

Comment:
State should be where we put things that we expect to change during the lifecycle of the app. keyMap should be pretty stable from day to day. This should be hard coded not in the state.